make/toolchain-android.cmake

```diff
@@ -47,9 +47,7 @@ set(CMAKE_SYSTEM_VERSION 1)
 if (NOT DEFINED TARGET_ABI)
   set(TARGET_ABI "arm-linux-androideabi")
 endif ()
-if (TARGET_ABI MATCHES "^arm")
-  set(CMAKE_SYSTEM_PROCESSOR arm)
-endif ()
+set(CMAKE_SYSTEM_PROCESSOR arm)
```

Comment:
This change doesn't look safe: there's also x86 android.  What is TARGET_ABI set to that isn't matching ^arm?